chain/consensus/mir/state_manager.go

```diff
@@ -598,32 +628,6 @@ func (sm *StateManager) firstEpochCheckpoint() (*Checkpoint, error) {
 	return ch, nil
 }
 
-// GetCheckpointPeriod returns the checkpoint period for the current epoch.
-//
-// The checkpoint period is computed as the number of validator times the
-// segment length.
-func (sm *StateManager) GetCheckpointPeriod() abi.ChainEpoch {
```

Comment:
Are we sure removing this doesn't break anything? We relied heavily on the `CheckpointPeriod` to determine the blocks to include in a checkpoint. If tests fail we should review this logic to see that we are doing it right.